WordPress/src/main/res/values/strings.xml

```diff
@@ -962,8 +959,11 @@
     </string-array>
     <string name="locale_picker_current_language">Current language:</string>
 
+    <!-- Experimental Features -->
+    <string name="experimental_features_screen_title">Experimental Features</string>
+
     <!-- Debug settings -->
-    <string name="preference_open_debug_settings" translatable="false">Debug settings</string>
+    <string name="preference_open_debug_settings" translatable="false">Debug Settings</string>
```

Comment:
Changed to titlecase to match all other buttons/screens.

Reply:
This is fine since we use titlecase elsewhere, but note that on Android [the recommendation](https://developers.google.com/style/capitalization) is to avoid unnecessary capitalization. This is something we're very inconsistent about.